src/include/platform/PlatformManager.h

```diff
@@ -109,6 +109,15 @@ class PlatformManager
      * other.
      */
     CHIP_ERROR InitChipStack();
+    void Shutdown();
```

Comment:
I _think_ these were sort of ordered in "order you might call them".  I assume moving them is just drive-by cleanup, not actually integral to this change?

Reply:
Function order cleanup (it didn't even match from declarations in class to inline implementations below) reverted for clarity in code review.